content/SCALE/SCALE22.12.md

```diff
@@ -79,7 +79,7 @@ It also implements fixes to pool status reporting, application options, reportin
 ### Component Versions
 
 TrueNAS SCALE is built from many different software components.
-This list has up to date information on which versions of Linux, ZFS, and NVIDIA drivers are included with this TrueNAS SCALE release.
+This list has up to date information on which versions of Linux, ZFS, and NVIDIA drivers included with this TrueNAS SCALE release.
```

Comment:
```suggestion
This list has up-to-date information on which versions of Linux, ZFS, and NVIDIA drivers are included with this TrueNAS SCALE release.
```

Reply:
fixed in my local copy to push in next commit.